src/tests/Tooltip.test.tsx

```diff
@@ -0,0 +1,37 @@
+import { screen, waitFor } from '@testing-library/react'
+import userEvent from '@testing-library/user-event'
+import React from 'react'
+import { beforeEach, describe, expect, it } from 'vitest'
+
+import TestRenderer from './TestRenderer'
+import TooltipPage from '../scenarios/Tooltip.scenario'
+
+describe('CheckBox component', () => {
+  beforeEach(() => {
+    sessionStorage.clear()
+  })
+
+  it('renders tootip when overing disabled button', async () => {
```

Comment:
Just a spelling mistake here:
```suggestion
  it('renders tootip when hovering disabled button', async () => {
```
and the same is needed on the below.

Reply:
Typo fixed